### PR TITLE
Feature/616 add first last names

### PR DESCRIPTION
**Pull request by mbrookeswebdev**
Implemented changes:

When the candidate's application form is viewed, Personal details should display first/last names if exist or a full name if they do not exist. Candidate's details page should also display first and last name and save updated details correctly (Personal Details component in the jac-kit was updated to reflect the changes).

Exercise-Application-JAC-Digital-Platform

### CODE REVIEW

File: src/views/Exercises/Applications/Application.vue [hunk 1]
```diff
@@ -180,7 +180,46 @@
               </h2>
 
               <dl class="govuk-summary-list">
-                <div class="govuk-summary-list__row">
+                <div
+                  v-if="application.personalDetails.firstName && application.personalDetails.lastName"
```
Comment (warrensearle) [inline src/views/Exercises/Applications/Application.vue:184]: Could perhaps put `application.personalDetails.firstName && application.personalDetails.lastName` in a computed as it is repeated three times in the template

**Review by warrensearle — COMMENTED** [review]
Tested on the preview URL and this is working well.
For bonus points I wonder if there's a way to allow the user editing `fullName` to be able to switch to editing `firstName` and `lastName` instead?

**Comment by HalcyonJAC** [comment]
@mbrookeswebdev 

The changes look ok to me but I am not really sure what the fullName vs. firstName and lastName split is really about to be honest. I am wondering though if they effect the reports that have been written at all? I ask because most of the reports reference `application.personalDetails.fullName`. For example: (see below):

image

File: src/views/Exercises/Applications/Application.vue (hunk 1 above)
Comment (mbrookeswebdev) [reply src/views/Exercises/Applications/Application.vue]: Done.

**Comment by mbrookeswebdev** [comment]
@HalcyonJAC We need to collect first/last names to enable the handover of data to the Judicial Office. Whilst we will be collecting this data, full names of candidates will still exist, so it should not affect your work.

**Review by warrensearle — APPROVED** [review]
This looks good to me. Tested locally 👍

**Comment by HalcyonJAC** [comment]
> @HalcyonJAC We need to collect first/last names to enable the handover of data to the Judicial Office. Whilst we will be collecting this data, full names of candidates will still exist, so it should not affect your work.

Gotcha! Thanks. :)

**Review by HalcyonJAC — APPROVED** [review]

**Pull request merged**